frontend/src/app/offers/offer-detail/offer-detail.component.ts

```diff
@@ -16,14 +19,31 @@ export class OfferDetailComponent implements OnInit {
   public offer$: Observable<Offer>;
   public djangoRoot = environment.djangoRoot;
   public getJoinViewUrl = this.offersService.getJoinViewUrl;
+  isUserOrgMember$: Observable<boolean>;
+  user$: Observable<User | null>;
```

Comment:
public

Reply:
Let's keep it like that for now, we do not have a defined styleguide for a project yet and I see there are different ideas of how should we write this...